package/gluon-geolocator/files/usr/lib/micron.d/geolocator

```diff
@@ -0,0 +1 @@
+*/5 * * * *     /lib/gluon/geolocator/geolocator
```

Comment:
How often would you expect someone to move their router? Five minutes is unusuable for a moving target, so that's probably not the use case. I'd say several hours (6/12/24) is more appropriate here.

Reply:
Could be a way, but then I would prefer a reboot script due to the interval time will be configurable over the configMode. But that also mean if some one changes the interval time via uci, he also need to rerun the reboot script manually...